src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -27,13 +27,13 @@ To get started, you will need:
 ## How to deploy a site with Git
 
 <Steps>
-1. Set up a new project on Cloudflare Pages.
+1. Push your code to your git repository (GitHub, GitLab).
 
-2. Push your code to your git repository (GitHub, GitLab).
+2. Log in to the Cloudflare dashboard and select your account.
 
-3. Log in to the Cloudflare dashboard and select your account in **Account Home** > **Compute (Workers)** > **Workers & Pages** > **Overview**.
+3. Go to **Account Home** > **Compute (Workers)** > **Workers & Pages**.
```

Comment:
I don't think you need to go to the `Account Home` menu, let me know if you have any other thoughts.

```suggestion
3. Go to **Compute (Workers)** > **Workers & Pages**.
```

Reply:
@jsparkdev You're right.  

In [Cloudflare Docs](https://developers.cloudflare.com/pages/get-started/git-integration/#connect-your-git-provider-to-pages):

> Pages offers support for [GitHub](https://github.com/) and [GitLab](https://gitlab.com/). To create your first Pages project:
>
> 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
> 2. In Account Home, select **Workers & Pages**.
> 3. Select **Create application** > **Pages** > **Connect to Git**.

It seems that after login/selecting an account, you are in Account Home. Do we need to point out this?

```suggestion
3. In Account Home, select **Compute (Workers)** > **Workers & Pages**.
```

Or just apply your change?

Reply:
@vrabe 

It seems right to describe it in the official documentation :)